s2i-datapower/README.md

```diff
@@ -0,0 +1,119 @@
+# Datapower s2i
+
+This project demonstrates how to create an s2i Datapower image to facilitate Datapower deployments in Openshift.
+
+![IBM Datapower](https://avatars3.githubusercontent.com/u/8836442?v=3&s=200 "IBM Datapower")
+
+## Table of content
+
+* [Create a Project ](#create-a-project)
+* [Create the s2i-datapower Image](#create-the-s2i-datapower-image)
+* [The Experimentation Use Case](#the-experimentation-use-case)
+* [The Immutable Use Case](#the-immutable-use-case)
+* [Examples](#examples)
+
+
+
+## Create a Project 
+
+This is the project win which we will run the below commands, you can use a different project.
+
+```
+oc new-project datapower
+```
+
+## Create the s2i-datapower Image
+
+This commands will create a build config whose outout will be the s2i-datapower.
+
+```
+oc new-build https://github.com/raffaelespazzoli/containers-quickstarts#datapower --context-dir=s2i-datapower --name=s2i-datapower --strategy=docker
```

Comment:
This should reference the [containers-quickstarts](https://github.com/redhat-cop/containers-quickstarts) repository